locales/en/default.json

```diff
@@ -37,7 +37,9 @@
     },
     "dob": {
       "required": "Please enter a date of birth",
-      "british-date": "Please enter a date of birth in the correct format"
+      "british-date": "Please enter a date of birth in the correct format",
+      "past": "Please enter a date of birth that is not in the future",
```

Comment:
"Please enter a date of birth that is in the past" ?
Maybe we should ask Louise what she thinks?